public-html/asset/scss/components/contactus.scss

```diff
@@ -0,0 +1,157 @@
+@import '../utils/variables.scss';
+
+.contact_us {
+    background-image: url("../images/drupal2 (copy).png");
```

Comment:
Can we rename this file to a better name instead of `drupal2 (copy).png`?

Reply:
changed